OpenAI-DotNet/OpenAIClient.cs

```diff
@@ -85,7 +85,10 @@ public OpenAIClient(OpenAIAuthentication openAIAuthentication = null, OpenAIClie
 
         private HttpClient SetupClient(HttpClient client = null)
         {
-            client ??= new HttpClient();
+            client ??= new HttpClient(new SocketsHttpHandler()
+            {
+                PooledConnectionLifetime = OpenAIClientSettings.PooledConnectionLifetime,
```

Comment:
how would this affect someone who doesn't set this when providing their own custom client?

Reply:
`PooledConnectionLifetime` cannot be set once the `HttpClient` has been created. It would be up to the person providing their own custom `HttpClient` set this value.

Reply:
What would be the impact if they did not set this?

Reply:
It sounds like DNS changes may not realized in high volume scenarios.

https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines#dns-behavior